### PR TITLE
[MLU] add mlu config support.

### PR DESCRIPTION
**Pull request by ShawnNew**
add mlu support in train.py, predict.py and eval.py

### CODE REVIEW

**Review by qili93 — DISMISSED** [review]
LGTM

File: predict.py [hunk 1]
```diff
@@ -149,6 +149,8 @@ def main(args):
         place = 'xpu'
     elif args.device == 'npu' and paddle.is_compiled_with_npu():
         place = 'npu'
+    elif args.device == 'mlu' and paddle.is_compiled_with_mlu():
```
Comment (juncaipeng) [inline predict.py:152]: 需要同步更新文件顶端的device注释

Comment (ShawnNew) [reply predict.py:152]: done

**Review by juncaipeng — APPROVED** [review]
LGTM

**Pull request merged**